### PR TITLE
[SOAR-844] - Revoke session with slack

### PR DESCRIPTION
**Pull request by jmcadams-r7**
## Description: 

This PR adds the Revoke_User_Session_in_Azure_AD_with_Slack workflow

This PR also updates Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams to use the correct github link in the yaml

## Validation: 
```
rmt-mbp-5357:Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams jmcadams$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 22.744ms
rmt-mbp-5357:Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams jmcadams$ cd ..
rmt-mbp-5357:workflows jmcadams$ cd Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/
rmt-mbp-5357:Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams jmcadams$ icon-validate .
[*] Validating workflow at .

[*] Running Integration Validators...
[*] Executing validator WorkflowDirectoryNameMatchValidator
[*] Executing validator WorkflowFilesValidator
[*] Executing validator WorkflowHelpValidator
[*] Executing validator WorkflowChangelogValidator
[*] Executing validator WorkflowVendorValidator
[*] Executing validator WorkflowVersionValidator
[*] Executing validator WorkflowExtensionValidator
[*] Executing validator WorkflowSupportValidator
[*] Executing validator WorkflowPNGHashValidator
[*] Executing validator WorkflowICONFileNameValidator
[*] Executing validator WorkflowScreenshotValidator
[*] Executing validator WorkflowTitleValidator
[*] Executing validator WorkflowDescriptionValidator
[*] Executing validator WorkflowNameValidator
[*] Executing validator WorkflowProfanityValidator
[*] Executing validator WorkflowHelpPluginUtilizationValidator
[*] Executing validator WorkflowICONFileValidator
[*] Workflow successfully validated!

----
[*] Total time elapsed: 22.875ms
```

## Workflow: 
workflow1

## Slack Flow: 
slack_flow1

### CODE REVIEW

File: workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/help.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ _There is no troubleshooting information at this time_
 
 # Version History
 
+* 1.0.1 - Update to fix github reference
```
Comment (jschipp-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/help.md:46]:
```suggestion
* 1.0.1 - Update to correct `source_url` reference in spec
```

File: workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/workflow.spec.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ products: ["insightconnect"]
 name: Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams
 title: "Revoke User Session in Azure AD with Microsoft Teams"
```
Comment (jschipp-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/workflow.spec.yaml:4]:
```suggestion
title: "Revoke User Session in Azure AD from Microsoft Teams"
```

File: workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/workflow.spec.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ hub_tags:
   keywords: [phishing, email, office365, slack, work_from_home, session]
```
Comment (jschipp-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/workflow.spec.yaml:12]:
```suggestion
  keywords: [phishing, email, office365, teams, chat, chatops, work_from_home]
```

File: workflows/Revoke_User_Session_in_Azure_AD_with_Slack/workflow.spec.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
+extension: workflow
+products: ["insightconnect"]
+name: Revoke_User_Session_in_Azure_AD_with_Slack
+title: "Revoke User Session in Azure AD with Slack"
```
Comment (jschipp-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_with_Slack/workflow.spec.yaml:4]:
```suggestion
title: "Revoke User Session in Azure AD from Slack"
```

File: workflows/Revoke_User_Session_in_Azure_AD_from_Slack/help.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+## Setup
+
+Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.
+
+### Usage
+
+This workflow uses the Slack Chat Ops trigger to monitor for a specific command. To trigger this workflow direct message the Rapid7 Insight Connect bot with:
+
+`!revoke-session user@example.com`
```
Comment (tylerslijboom-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_from_Slack/help.md:24]:
```suggestion
`revoke-session user@example.com`
```
I believe Slack items dont' have !

File: workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/workflow.spec.yaml [hunk 6]
```diff
@@ -1,21 +1,21 @@
...
 support: rapid7
 status: []
 hub_tags:
   use_cases: [threat_detection_and_response]
-  keywords: [phishing, email, office365, slack, work_from_home, session]
+  keywords: [phishing, email, office365, teams, chat, chatops, work_from_home]
   features: []
 resources:
-  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Purge_Office_365_Emails_with_Slack
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Purge_Office_365_Revoke_User_Session_in_Azure_AD_with_Slack
```
Comment (tylerslijboom-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_with_Microsoft_Teams/workflow.spec.yaml:15]: Doesn't this line have to match the folder name?

File: workflows/Revoke_User_Session_in_Azure_AD_from_Slack/help.md (hunk 5 above)
Comment (jmcadams-r7) [reply workflows/Revoke_User_Session_in_Azure_AD_from_Slack/help.md:24]: They don't, I forgot to update the help. Good catch.

File: workflows/Revoke_User_Session_in_Azure_AD_from_Slack/help.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+|Plugin|Version|Count|
+|----|----|--------|
+|Python 3 Script|2.0.1|1|
+|String Operations|1.2.1|1|
+|Azure AD Admin|2.2.0|1|
+
+## Troubleshooting
+
+_There is no troubleshooting information at this time_
```
Comment (jschipp-r7) [inline workflows/Revoke_User_Session_in_Azure_AD_from_Slack/help.md:38]:
```suggestion
_There is no troubleshooting information at this time._
```

**Pull request merged**